documents/DeveloperGuide/ShaderGeneration.md

```diff
@@ -1,12 +1,16 @@
 # Shader Generation
 
 ## 1.1 Scope
-Shader generation features are implemented as a core part of MaterialX. Most
-features are contained in the [MaterialXGenShader](/source/MaterialXGenShader) shared library, but some features are part of the [MaterialXCore](/source/MaterialXCore) library as well.
+Shader generation features are implemented as a core part of MaterialX. Core
+feature support is contained in the [MaterialXGenShader](/source/MaterialXGenShader) shared library, with some features part of the [MaterialXCore](/source/MaterialXCore) library.
```

Comment:
Maybe we could remove the last part of this sentence now, after the comma. The fact that we did updates to MaterialXCore in order to support our shader generation is not that relevant for a user at this point.